srai/regionizers/_spherical_voronoi.py

```diff
@@ -243,7 +245,11 @@ def _create_region(
 
 
 def generate_voronoi_regions(
-    seeds: List[Point], max_meters_between_points: int, allow_multiprocessing: bool
+    seeds: List[Point],
+    max_meters_between_points: int,
+    allow_multiprocessing: bool,
+    num_of_multiprocessing_workers: Optional[int] = None,
```

Comment:
I'm thinking you could maybe replace these two with a single parameter - num_of_multiprocessing_workers? You could make it similar to what sklearns _num_jobs_ so -1 means all, None means 1. Then not using this parameter effectively means `allow_multiprocessing=False` and `num_of_multiprocessing_workers=1` which I think is what most users expect when using a library. It simplifies the API. You could also make it so that the default is -1 meaning all cpus if that's what makes sense here.
I also understand the idea behind `CPU_COUNT-1` but I wonder if that's what's expected. Maybe using all by default would feel more _natural_?

Reply:
+1

Reply:
You're right, suggestions applied